server/routes/api/delete/deleteUserDBHandler.ts

```diff
@@ -0,0 +1,44 @@
+import db from "../../../utils/db/db";
+import { UserDelete } from "./deleteUserTypes";
+
+export default class DBHandler {
+    async getParent(parentId: number): Promise<UserDelete[]> {
+        const parentQuery = `SELECT end_timestamp
+                             FROM parents
+                             WHERE parent_id = $1 
+                             LIMIT 1`;
```

Comment:
I think you should add `ORDER BY` to make sure you get the latest timestamp and not a randomize timestamp [ref](https://bipp.io/sql-tutorial/basic-sql/sql-order-by-limit-clauses/)

Reply:
here specifically we ask for one entry so i think it will be useless. BUT some other queries should add it. 
for example: getAllBabysitters etc.
will do

Reply:
Anyway parent_id is unique so it will always return 1 row. No need for limit 1